INSTALL.md

```diff
@@ -118,6 +120,63 @@ If the desired python version is not installed, you can install it using
 [Xcode Command Line Tools](https://stackoverflow.com/questions/9329243/how-to-install-xcode-command-line-tools)
 needs to be installed for development.
 
+<a name="Apple-M1-Build-Dependencies"></a>
+##### Apple M1
+- Command line tools
+    ```bash
+    xcode-select --install
+    ```
```

Comment:
this is necessary for non M1 system as well. Should be moved in the previous section or could be  removed?